app/controllers/projects/classifies_controller.rb

```diff
@@ -7,6 +7,11 @@ def show
 
     @info_request =
       @project.info_requests.where(awaiting_description: true).sample
+
+    unless @info_request
+      msg = _('There are no requests to classify right now. Great job!')
+      redirect_to @project, notice: msg
```

Comment:
Need a `return` after this line for when we integrate https://github.com/mysociety/alaveteli/pull/5665/files#diff-3ee52fe46c4a210851d4d5beba970a93R11

Reply:
Fixed in e2300bf89900f549dff274a488e7b43c0d97214a